### DISCUSSION TITLE
Flows as Code ?

### DISCUSSION THREAD

**Discussion by MentalGear** [directus/directus#16634] [Archive]
I spend a few days wiring around with the Flows UI. It's a nice idea for small things, but annoyingly cumbersome to debug for a bit more serious/big workflows, and as a Dev, I would much more prefer a code-only solution that can be version-controlled, tested and probably debugged.

Yet, the direct integration with Directus along with low-latency access to the DB makes Flows attractive.

Is there a way to have a code-only version of Flows ?
If not, I'd like to add a __feature request__

**Comment by br41nslug** [comment]
https://docs.directus.io/extensions/introduction.html

**Reply by MentalGear** [reply]
Yes, I thought about this, but there's currently no way to use extensions with directus cloud.
(If you are aware of another no-hazzle directus host, please let me know)

Regarding Flows, it would be great if there was a UI/code View toggle.
